bigtable-hbase-1.x-parent/hbase-client-1x-compat/pom.xml

```diff
@@ -0,0 +1,527 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--Based on hbase-shaded-client 1.7.2, with more transitive exclusions and
+updated transitive versions -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>bigtable-hbase-1.x-parent</artifactId>
+        <groupId>com.google.cloud.bigtable</groupId>
+        <version>2.7.3-SNAPSHOT</version> <!-- {x-version-update:bigtable-client-parent:current} -->
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>hbase-client-1x-compat</artifactId>
```

Comment:
nit - rename to `1.x` here and in the folder name

Reply:
done